### PR TITLE
Remove now-unused PermissionScope property from message models

### PR DESCRIPTION
**Pull request by wtritch**
Final cleanup of the report-processor.  This removes the vestigial PermissionScope from the DTO model instances.

### CODE REVIEW

File: webapp/src/main/java/org/opentestsystem/rdw/reporting/report/DefaultExamReportService.java [hunk 1]
```diff
@@ -34,19 +29,17 @@
 
     @Override
     public Report createReport(@NotNull final User user, @NotNull final StudentExamReportRequest request) {
-        return createReport(user, request, null);
+        return createReportInternal(user, request);
     }
```
Comment (wtritch) [inline webapp/src/main/java/org/opentestsystem/rdw/reporting/report/DefaultExamReportService.java]: I debated removing the three methods and just making the abstract parameter method public... thoughts?

Comment (esotrope) [reply webapp/src/main/java/org/opentestsystem/rdw/reporting/report/DefaultExamReportService.java]: Yeah Id go with that. The method bodies are all the same now.

Comment (wtritch) [reply webapp/src/main/java/org/opentestsystem/rdw/reporting/report/DefaultExamReportService.java]: hm... except I like the different authorities on the interface based upon the request type...
```
/**
     * Creates an individual student exam report
     *
     * @param user    the user requesting the report
     * @param request settings which shape the content of the report
     * @return the created report handle
     */
    @PreAuthorize("hasAnyAuthority('PERM_INDIVIDUAL_PII_READ', 'PERM_GROUP_PII_READ')")
    Report createReport(@NotNull User user, @NotNull StudentExamReportRequest request);

    /**
     * Creates a school grade scoped student exam report
     *
     * @param user    the user requesting the report
     * @param request settings which shape the content of the report
     * @return the created report handle
     */
    @PreAuthorize("hasAuthority('PERM_INDIVIDUAL_PII_READ')")
    Report createReport(@NotNull User user, @NotNull SchoolGradeExamReportRequest request);

    /**
     * Creates a group scoped student exam report
     *
     * @param user    the user requesting the report
     * @param request settings which shape the content of the report
     * @return the created report handle
     */
    @PreAuthorize("hasAuthority('PERM_GROUP_PII_READ') and #request.groupGrant != null")
    Report createReport(@NotNull User user, @NotNull GroupExamReportRequest request);
```

Comment (esotrope) [reply webapp/src/main/java/org/opentestsystem/rdw/reporting/report/DefaultExamReportService.java]: Yeah that's a deal breaker we'll want to keep that.

**Pull request merged**